client/webpack/webpackConfig.dev.js

```diff
@@ -46,7 +46,7 @@ module.exports = {
 			{
 				test: /\.(js|jsx|ts|tsx)$/,
 				include: [resolve(__dirname, '../'), resolve(__dirname, '../../utils')],
-				use: 'babel-loader',
+				use: 'ts-loader',
```

Comment:
**Important change** — output files into `dist/client` directory and use `ts-loader` to transform TypeScript source. The same change was made for the prod webpack config below.

Reply:
see ya, Babel!